### PR TITLE
Add `.gitattributes` file

### PR DESCRIPTION
**Pull request by alamirault**
Hello,

This PR add `.gitattributes` with `export-ignore` in order to reduce vendor size

More info here:
https://php.watch/articles/composer-gitattributes#export-ignore

### CODE REVIEW

**Comment by aik099** [comment]
Might be related #69 .

**Comment by alamirault** [comment]
With no explanation on other PR, it's complicated to know why it was refused :smile:

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/github export-ignore
```
Comment (reedy) [inline .gitattributes:1]: .github

**Comment by reedy** [comment]
It's also a dupe of https://github.com/hamcrest/hamcrest-php/pull/76

**Comment by alamirault** [comment]
Close as https://github.com/hamcrest/hamcrest-php/pull/76 has been merged

**Pull request closed without merging**